src/pages/ProjectSettings.tsx

```diff
@@ -30,10 +30,10 @@ import { getTabTitle } from "./projectSettings/getTabTitle";
 import { ProjectSettingsTabs } from "./projectSettings/Tabs";
 import { ProjectType } from "./projectSettings/tabs/utils";
 
-const { isProduction } = environmentalVariables;
+const { isDevelopment } = environmentalVariables;
 const { validateObjectId } = validators;
 
-const disablePage = isProduction();
+const disablePage = !isDevelopment();
```

Comment:
`isDevelopment` will not allow this page to be viewable in staging. `isProduction` and `isDevelopment` only indicate if we have a compiled build or a build running on a dev server. I would recommend doing something akin to what we do with `isBeta` having an environmental variable that is added to the compile step.